contracts/utils/Votes.sol

```diff
@@ -0,0 +1,210 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./Context.sol";
+import "./Counters.sol";
+import "./Checkpoints.sol";
+import "./cryptography/draft-EIP712.sol";
+
+/**
+ * @dev Voting operations.
```

Comment:
We need more extensive documentation that explains what the contract is and how it's used. Important points to mention: when using this module, the derived contract must implement `_getDelegatorVotes`, and can use `_moveVotingPower` when a delegator's voting power is changed.